primefaces/src/main/java/org/primefaces/component/api/UITable.java

```diff
@@ -203,6 +204,18 @@ default void updateFilterByValuesWithFilterRequest(FacesContext context, Map<Str
                 filterValue = params.get(valueHolderClientId);
             }
 
+            if (filterMeta.isShortFieldNotation()) {
```

Comment:
can you add a comment here, why its only done when #isShortFieldNotiation?

everything else looks good -> +1